packages/nextra-theme-blog/src/posts-layout.tsx

```diff
@@ -29,24 +31,27 @@ export const PostsLayout = ({ children }: { children: ReactNode }) => {
     const description = post.frontMatter?.description
 
     return (
-      <div key={post.route}>
-        <h3>
+      <div key={post.route} className="post-item">
+        <h3 className="post-item-heading">
           <Link href={post.route} passHref>
-            <a className="!no-underline">{postTitle}</a>
+            <a className="post-item-title !no-underline">{postTitle}</a>
```

Comment:
```suggestion
            <a className="!no-underline">{postTitle}</a>
```
`.post-item > h3 > a`